src/reducers/dependencies.reducer.js

```diff
@@ -120,6 +126,91 @@ export default (state: State = initialState, action: Action) => {
       });
     }
 
+    case ADD_DEPENDENCIES_START: {
+      const { projectId, dependencies } = action;
+
+      return produce(state, draftState => {
+        // this action name is a bit of a misnomer, since this block
+        // is used for both adding AND updating dependencies (otherwise,
+        // we wouldn't be able to batch installs/updates with a single
+        // `npm install` command
+
+        dependencies.forEach(dependency => {
+          draftState[projectId][dependency.dependencyName] = {
+            name: dependency.dependencyName,
+            status: dependency.updating ? 'updating' : 'installing',
```

Comment:
Following up from earlier comment... would anything happen if we always set the status to `installing`, even if it was actually updating?

Reply:
What's the current flow for updating dependencies? Does the dependency detail pane display a status notification similar to the fish animation it shows during install, or does it remain unchanged until the update is finished?